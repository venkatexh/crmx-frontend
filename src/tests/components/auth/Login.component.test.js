import { shallow } from "enzyme";
import React from "react";
import LoginComponent from "../../../components/auth/Login.component";

it("Login component snapshot", () => {
  expect(shallow(<LoginComponent />)).toMatchSnapshot();
});
