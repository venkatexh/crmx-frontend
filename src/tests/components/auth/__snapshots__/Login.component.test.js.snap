// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component snapshot 1`] = `ShallowWrapper {}`;
